features/fixtures/TestApp/config/config.yml

```diff
@@ -45,11 +45,11 @@ fos_user:
     firewall_name:    api
     service:
         user_manager: api.user_manager
-    user_class:       "Dunglas\ApiBundle\Tests\Behat\TestBundle\Entity\User"
+    user_class:       "Dunglas\\ApiBundle\\Tests\\Behat\\TestBundle\\Entity\\User"
```

Comment:
+1 I'm using single quotes everywhere in yaml. It also resolves the issue with `\P` that is disappearing.